From 36f5ca6c59024363169f82642596e791b3733cd6 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Wed, 12 Oct 2011 13:44:31 +0000
Subject: Purge hardcoded -DG_DISABLE_DEPRECATED

Especially now that deprecations are proper gcc warnings, no need
to hardcode this.
---
diff --git a/client/Makefile.am b/client/Makefile.am
index 925300b..a81db1d 100644
--- a/client/Makefile.am
+++ b/client/Makefile.am
@@ -5,7 +5,7 @@ INCLUDES = -I$(top_srcdir) \
 	-I$(top_srcdir)/metadata \
 	-I$(top_srcdir)/common \
 	$(GLIB_CFLAGS) $(DBUS_CFLAGS) \
-	-DG_LOG_DOMAIN=\"GVFS\" -DG_DISABLE_DEPRECATED \
+	-DG_LOG_DOMAIN=\"GVFS\" \
 	-DGVFS_MODULE_DIR=\"$(libdir)/gvfs/modules\"	\
 	-DGVFS_LOCALEDIR=\""$(localedir)"\"     \
 	-DDBUS_API_SUBJECT_TO_CHANGE
@@ -74,7 +74,7 @@ gvfs_fuse_daemon_CFLAGS = \
 	-I$(top_srcdir) \
         -I$(top_srcdir)/common \
         $(GLIB_CFLAGS) $(DBUS_CFLAGS) $(FUSE_CFLAGS) \
-        -UG_LOG_DOMAIN -DG_LOG_DOMAIN=\"GVFS-FUSE\" -DG_DISABLE_DEPRECATED \
+        -UG_LOG_DOMAIN -DG_LOG_DOMAIN=\"GVFS-FUSE\" \
         -DDBUS_API_SUBJECT_TO_CHANGE
 
 gvfs_fuse_daemon_LDADD = \
diff --git a/common/Makefile.am b/common/Makefile.am
index fefd441..5d5a54e 100644
--- a/common/Makefile.am
+++ b/common/Makefile.am
@@ -5,7 +5,7 @@ lib_LTLIBRARIES=libgvfscommon.la
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/gvfs \
 	-I$(top_srcdir)/daemon \
 	$(GLIB_CFLAGS) $(DBUS_CFLAGS) $(BLURAY_CFLAGS) \
-	-DG_LOG_DOMAIN=\"GVFS\" -DG_DISABLE_DEPRECATED \
+	-DG_LOG_DOMAIN=\"GVFS\" \
 	-DDBUS_API_SUBJECT_TO_CHANGE
 
 libgvfscommon_la_SOURCES = 	\
diff --git a/daemon/Makefile.am b/daemon/Makefile.am
index 6ca840b..b19ee89 100644
--- a/daemon/Makefile.am
+++ b/daemon/Makefile.am
@@ -16,8 +16,7 @@ INCLUDES =					\
 	-DG_UDEV_API_IS_SUBJECT_TO_CHANGE	\
 	-DLIBEXEC_DIR=\"$(libexecdir)/\" 	\
 	-DMOUNTABLE_DIR=\"$(mountdir)/\" 	\
-	-DGVFS_LOCALEDIR=\""$(localedir)"\"     \
-	-DG_DISABLE_DEPRECATED
+	-DGVFS_LOCALEDIR=\""$(localedir)"\"
 
 noinst_LTLIBRARIES=libdaemon.la
 
@@ -496,7 +495,7 @@ gvfsd_afp_browse_CPPFLAGS = \
 gvfsd_afp_browse_LDADD = \
 	$(libraries) \
 	$(LIBGCRYPT_LIBS)
-	
+
 gvfsd_afp_SOURCES = \
 	gvfsbackendafp.c gvfsbackendafp.h \
 	daemon-main.c daemon-main.h \
diff --git a/metadata/Makefile.am b/metadata/Makefile.am
index becf13e..d3f25be 100644
--- a/metadata/Makefile.am
+++ b/metadata/Makefile.am
@@ -23,7 +23,7 @@ INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/common \
 	$(LIBXML_CFLAGS) $(GLIB_CFLAGS) $(DBUS_CFLAGS) \
 	$(UDEV_CFLAGS) \
 	-DGVFS_LOCALEDIR=\""$(localedir)"\"     \
-	-DG_LOG_DOMAIN=\"GVFS\" -DG_DISABLE_DEPRECATED \
+	-DG_LOG_DOMAIN=\"GVFS\" \
 	-DDBUS_API_SUBJECT_TO_CHANGE
 
 libmetadata_la_SOURCES = 		\
diff --git a/monitor/afc/Makefile.am b/monitor/afc/Makefile.am
index a823080..e524481 100644
--- a/monitor/afc/Makefile.am
+++ b/monitor/afc/Makefile.am
@@ -19,7 +19,6 @@ gvfs_afc_volume_monitor_CFLAGS = \
 	$(WARN_CFLAGS) \
 	-DGIO_MODULE_DIR=\"$(GIO_MODULE_DIR)\" \
 	-DGVFS_LOCALEDIR=\"$(localedir)\" \
-	-DG_DISABLE_DEPRECATED \
 	$(NULL)
 
 gvfs_afc_volume_monitor_LDADD = \
diff --git a/monitor/gdu/Makefile.am b/monitor/gdu/Makefile.am
index c9c9195..4945445 100644
--- a/monitor/gdu/Makefile.am
+++ b/monitor/gdu/Makefile.am
@@ -22,7 +22,6 @@ gvfs_gdu_volume_monitor_CFLAGS =		\
 	$(DBUS_CFLAGS)                          \
 	-DGIO_MODULE_DIR=\"$(GIO_MODULE_DIR)\"	\
 	-DGVFS_LOCALEDIR=\""$(localedir)"\"	\
-	-DG_DISABLE_DEPRECATED			\
 	-DGDU_API_IS_SUBJECT_TO_CHANGE		\
 	$(NULL)
 
diff --git a/monitor/gphoto2/Makefile.am b/monitor/gphoto2/Makefile.am
index 80fc424..10b25a7 100644
--- a/monitor/gphoto2/Makefile.am
+++ b/monitor/gphoto2/Makefile.am
@@ -39,7 +39,6 @@ gvfs_gphoto2_volume_monitor_CFLAGS =		\
 	$(GPHOTO2_CFLAGS)			\
 	-DGIO_MODULE_DIR=\"$(GIO_MODULE_DIR)\"	\
 	-DGVFS_LOCALEDIR=\""$(localedir)"\"	\
-	-DG_DISABLE_DEPRECATED			\
 	-DG_UDEV_API_IS_SUBJECT_TO_CHANGE	\
 	$(NULL)
 
diff --git a/monitor/hal/Makefile.am b/monitor/hal/Makefile.am
index 2fd3e6a..da8fc7b 100644
--- a/monitor/hal/Makefile.am
+++ b/monitor/hal/Makefile.am
@@ -33,7 +33,6 @@ gvfs_hal_volume_monitor_CFLAGS =		\
 	$(HAL_CFLAGS)                           \
 	-DGIO_MODULE_DIR=\"$(GIO_MODULE_DIR)\"	\
 	-DGVFS_LOCALEDIR=\""$(localedir)"\"	\
-	-DG_DISABLE_DEPRECATED			\
 	$(NULL)
 
 gvfs_hal_volume_monitor_LDFLAGS =	\
diff --git a/monitor/proxy/Makefile.am b/monitor/proxy/Makefile.am
index 86a1efe..bd3b451 100644
--- a/monitor/proxy/Makefile.am
+++ b/monitor/proxy/Makefile.am
@@ -25,7 +25,6 @@ libgioremote_volume_monitor_la_CFLAGS =					\
 	-DGIO_MODULE_DIR=\"$(GIO_MODULE_DIR)\"				\
 	-DREMOTE_VOLUME_MONITORS_DIR=\"$(remote_volume_monitorsdir)\" 	\
 	-DGVFS_LOCALEDIR=\""$(localedir)"\"				\
-	-DG_DISABLE_DEPRECATED						\
 	$(NULL)
 
 libgioremote_volume_monitor_la_LDFLAGS = 				\
@@ -53,7 +52,6 @@ libgvfsproxyvolumemonitordaemon_noin_la_CFLAGS =		\
 	$(GDU_CFLAGS)                           		\
 	-DG_LOG_DOMAIN=\"GVFS-RemoteVolumeMonitorDaemon\"	\
 	-DGVFS_LOCALEDIR=\""$(localedir)"\"			\
-	-DG_DISABLE_DEPRECATED					\
 	$(NULL)
 
 libgvfsproxyvolumemonitordaemon_noin_la_LIBADD  =     		\
diff --git a/programs/Makefile.am b/programs/Makefile.am
index 532f347..f99eaef 100644
--- a/programs/Makefile.am
+++ b/programs/Makefile.am
@@ -4,8 +4,7 @@ INCLUDES =					\
 	-I$(top_srcdir)				\
 	-I$(top_builddir)			\
 	$(GLIB_CFLAGS) 				\
-	-DGVFS_LOCALEDIR=\""$(localedir)"\"	\
-	-DG_DISABLE_DEPRECATED
+	-DGVFS_LOCALEDIR=\""$(localedir)"\"
 
 libraries =					\
 	$(GLIB_LIBS)
diff --git a/test/Makefile.am b/test/Makefile.am
index 70c0cb7..fb6c4b2 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -3,8 +3,7 @@ NULL =
 AM_CFLAGS =                       \
 	-I$(top_srcdir)           \
 	-I$(top_builddir)         \
-	$(GLIB_CFLAGS)            \
-	-DG_DISABLE_DEPRECATED
+	$(GLIB_CFLAGS)
 
 AM_LDFLAGS =                           \
 	$(GLIB_LIBS)
--
cgit v0.9.0.2
